src/Http/Controllers/User/PasswordController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Statamic\Http\Controllers\User;
+
+use Statamic\Facades\User;
+use Statamic\Http\Requests\UserPasswordRequest;
+
+class PasswordController
+{
+    public function __invoke(UserPasswordRequest $request)
+    {
+        $user = User::current();
+
+        $user->password($request->password);
+
+        $user->save();
+
+        return $this->successulResponse();
```

Comment:
typo `successFulResponse`

Reply:
Thanks. It even told me. Whoops.

![CleanShot 2024-05-16 at 09 51 24](https://github.com/statamic/cms/assets/105211/5af9176f-0874-447e-9c21-2f94d69ccaf7)